Scripts/script-ADGetComputer.yml

```diff
@@ -3,33 +3,21 @@ commonfields:
   version: -1
 name: ADGetComputer
 script: |-
-  # Optional arguments and default values
-  attrs = 'name,displayname,memberOf'
-  if demisto.get(demisto.args(), 'attributes'):
-      attrs = "," + demisto.args()['attributes']
-  userDN = ''
-  resp = ''
-  if demisto.get(demisto.args(), 'dn'):
-      userDN = demisto.args()['dn']
-  elif demisto.get(demisto.args(), 'name'):
-      resp = demisto.executeCommand( 'ad-search', { 'filter' : "(&(objectCategory=Computer)(name=" + demisto.args()['name'] + "))" } )
-  else:
-      demisto.results( { 'Type' : entryTypes['error'], 'ContentsFormat' : formats['text'], 'Contents' : 'You must provide either dn or name as argument!' } )
-      sys.exit(0)
-  if type(resp)==list and len( [ r for r in resp if isError(r) ] ) > 0 :
-      demisto.results( { 'Type' : entryTypes['error'], 'ContentsFormat' : formats['text'], 'Contents' : 'Error returned by ad command: ' + r['Contents'] } )
-      sys.exit(0)
-  if type(resp)==list and len(resp)==1 and type(resp[0])==dict and 'Contents' in resp[0] and type(resp[0]['Contents'])==list and len(resp[0]['Contents'])==1 and type(resp[0]['Contents'][0])==dict and 'dn' in resp[0]['Contents'][0]:
-      userDN = resp[0]['Contents'][0]['dn']
-  elif resp[0]['Contents'] == 'No results':
-      demisto.results( { 'Type' : entryTypes['error'], 'ContentsFormat' : formats['text'], 'Contents' : 'Item not found.' } )
-      sys.exit(0)
-  else:
-      demisto.results( { 'Type' : entryTypes['error'], 'ContentsFormat' : formats['text'], 'Contents' : 'Unexpected output from ad command.' } )
-      sys.exit(0)
-  if userDN:
-      filterstr = r"(&(objectClass=User)(distinguishedName=" + userDN + "))"
-      resAD = demisto.executeCommand( 'ad-search', { 'filter' : filterstr, 'attributes' : attrs } )
+  def createEndpointEntities(t,attrs):
+      endpoints = []
+      for l in t:
+          endpoint = {}
+          endpoint['Type'] = 'AD'
+          endpoint['ID'] = demisto.get(l,'dn')
+          endpoint['Username'] = demisto.get(l,'name')
```

Comment:
My mistake here is `Username` instead of `Hostname`.
what other attributes need to be change (I think type=`AD` is still relevant)